commit 517b12f2a0d9f9e6de680b30242ec6432d2af7f7
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:20:43 2010 -0700

    Staging: brcm80211: remove free_netdev wrapper
    
    It's incorrect, call the real function.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/linuxver.h
+++ b/drivers/staging/brcm80211/include/linuxver.h
@@ -63,10 +63,6 @@ typedef irqreturn_t(*FN_ISR) (int irq, v
 #define	PCI_SAVE_STATE(a, b)	pci_save_state(a)
 #define	PCI_RESTORE_STATE(a, b)	pci_restore_state(a)
 
-#ifndef HAVE_FREE_NETDEV
-#define free_netdev(dev)		kfree(dev)
-#endif
-
 /* suspend args */
 #define DRV_SUSPEND_STATE_TYPE pm_message_t
 
